src/app/components/core/south/add-service-wizard/add-service-wizard.component.html

```diff
@@ -30,35 +30,50 @@
             </div>
             <div class="field-body">
               <div class="select is-multiple">
-                <select multiple name="type" #selectedPlugin formControlName="plugin" (change)="getDescription(selectedPlugin.value)"
```

Comment:
` <small class="no-rec">No south plugin found</small>`
should be `<small class="no-rec">No south plugin installed</small>`

To differentiate the message installed vs available!